### PR TITLE
Machine-readable data section in dataset page

### PR DESCRIPTION
**Pull request by aFigueroadelatorre**
Added new snippet "mrd_section.html" with the section, and assets to implement "popover" pop-up for information. Snippet is visible in the side section of the dataset page (package/read_base.html template used for this) when DCAT extension is installed and enabled.

### CODE REVIEW

File: ckanext/ukdstheme/assets/js/ukdstheme.js
Comment (paulchauch) [inline ckanext/ukdstheme/assets/js/ukdstheme.js:1]: would be better if the HTML content can be passed in instead of written in

**Review by paulchauch — APPROVED** [review]
It looks fine to me

**Review by CamHurst — APPROVED** [review]
All looks good to me also. though i do agree if possible to move that block of HTML content to be passed in rather than being written in right in the middle of the code. perhaps at least a variable declared at the start of the code instead so it could be easily changed if needs be?

**Comment by aFigueroadelatorre** [comment]
Next possible fixes:

- Make the HTML content to be passed in rather than being written inside the JS code.

**Pull request merged**